content/articles/understanding-java-inheritance-and-polymorphism/index.md

```diff
@@ -0,0 +1,336 @@
+### Introduction
+ 
+Java is an object-oriented programming language since it provides Java developers with the ability to implement objects as real-life entities. OOP is a programming paradigm that is built around objects and their properties (attributes) and behaviors (methods). So, generally, software design using object-oriented principles is focused on objects and operations on them. Amongst many features of object-oriented programming is the fact that it encourages code reusability and extensibility. We will explain these two terms in detail later in this article.
```

Comment:
You need to explain that this article will be focused solely on polymorphism and inheritance. This introduction only talks about OOP in general, making it sound like it's on OOP in general. A good intro should give a general idea about what the guide will talk about.

Reply:
Corrections made.